package.json

```diff
@@ -59,7 +59,7 @@
     }
   ],
   "dependencies": {
-    "@adobe/aep-rules-engine": "^2.0.1",
+    "@adobe/aep-rules-engine": "file:../aepsdk-rulesengine-typescript",
```

Comment:
This is for local e2e test, will add dependency once rule engine changes are approved &  published.